aws/cloudformation/cloud_formation_stack.yml.erb

```
@@ -2,6 +2,18 @@
 AWSTemplateFormatVersion: 2010-09-09
 Description: AWS CloudFormation Template for Code.org application
 Parameters:
+  BaseDomainName:
+    Type: String
+    Default: code.org
+    Description: Base domain name.
+  DashboardSubDomainName:
+    Type: String
+    Default: !Sub "<%=rack_env?(:production) ? 'studio' : 'studio-${AWS::StackName}' %>"
```

I think this should actually be

```suggestion
    Default: !Sub "<%=rack_env?(:production) ? 'studio' : '${AWS::StackName}-studio' %>"
```

---

Ouch. Yes. Fixed!